wagtail/admin/templates/wagtailadmin/home/upgrade_notification.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n wagtailcore_tags wagtailadmin_tags %}
 {% wagtail_version as current_version %}
 <div
-    class="w-panel-upgrade-notification panel nice-padding"
+    class="w-panel-upgrade-notification panel"
```

Comment:
as noted above - if there are other panels that no longer need this (i.e add double padding) we can probably make this same change there.